internal/quickstart/show_sdk_instructions.go

```diff
@@ -80,18 +70,27 @@ func NewShowSDKInstructionsModel(
 		environment:        environment,
 		flagsClient:        flagsClient,
 		flagKey:            flagKey,
-		help:               h,
+		help:               help.New(),
 		helpKeys: keyMap{
 			Back:       BindingBack,
 			CursorDown: BindingCursorDown,
 			CursorUp:   BindingCursorUp,
 			Quit:       BindingQuit,
 		},
-		sdkKind:  sdkKind,
-		spinner:  s,
-		url:      url,
-		viewport: vp,
+		sdkKind: sdkKind,
+		spinner: s,
+		url:     url,
 	}
+
+	vp := viewport.New(
+		lipgloss.Width(m.headerView()),
```

Comment:
should we use the window width instead? this could also cut down on the amount of scrolling/would reduce code getting cut off at weird spots when copy/pasting over.

Reply:
Updated